### ISSUE TITLE
Unused Variable for import used in object literal property value shorthand

### ISSUE THREAD

**Issue by unional** [Microsoft/TypeScript#7280]
**TypeScript Version:** 
1.8.2

**Code**

```ts
import funcA from './funcA'; // Unused Variable 'funcA'

// either
export default {
  funcA
}

// or
let a = {
  funcA
};
```

**Comment by mhegazy** [comment]
I can not piece together the issue here. is this unused variable in the generated JS? or is it flagged, where it should not? what is the actual behavior, and how is this different from what you expect?

**Comment by unional** [comment]
It is flagged by tslint through language service as a warning:
![image](https://cloud.githubusercontent.com/assets/3254987/13404129/248b84ec-decd-11e5-972e-4fe36eb028c8.png)
![image](https://cloud.githubusercontent.com/assets/3254987/13404189/5bd65b66-decd-11e5-9726-b4cc2bd2642b.png)

The warning is gone when shorthand is not used:
![image](https://cloud.githubusercontent.com/assets/3254987/13404204/6f3d8fda-decd-11e5-8fd9-04866d76f85b.png)

**Comment by RyanCavanaugh** [comment]
@unional Please log TSLint bugs on the TSLint issue tracker

**Comment by unional** [comment]
Ok. Thanks. Just about to do so

**Comment by unional** [comment]
But could this be related to https://github.com/Microsoft/TypeScript/issues/3688?
